packages/yoroi-extension/chrome/extension/background.js

```diff
@@ -939,11 +939,12 @@ chrome.runtime.onMessage.addListener(
   // Returning `true` is required by Firefox, see:
```

Comment:
Might want to change the comment as well to be consistent with the code?

Reply:
ah heck yes